Mixer.py

```diff
@@ -10,7 +10,10 @@ def __init__(self):
         self.columns = []
         self.delimiter = ','
 
-    def add_column(self, source_list):
+    def add_column(self, source_list,name=None):
+        if name is not None:
+            for i in range(len(source_list)):
+                source_list[i]='"'+name+'"'+':'+'"'+source_list[i]+'"'
```

Comment:
* use dictionary instead of list
* do not update other parameter (source_list) in the method. if you need a modified copy of the list use map() function or similar that will create new collection
* do not concatenate strings using "+". Use "string".format() or "%s" % value form. More here: https://pyformat.info